downloadDirectory/pom.xml

```diff
@@ -108,13 +109,31 @@
 
 	<build>
 		<plugins>
+			<plugin>
+				<artifactId>exec-maven-plugin</artifactId>
+				<groupId>org.codehaus.mojo</groupId>
+				<version>1.6.0</version>
+				<executions>
+					<execution>
+						<id>Build PathwayExchange</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>exec</goal>
+						</goals>
+						<configuration>
+							<executable>build_pathway_exchange.sh</executable>
```

Comment:
This looks intriguing!  Does it run the shell script as a build pre-requisite?

Reply:
Oops... this POM wasn't supposed to be checked in. I will remove these changes.

Reply:
But yes, you can execute shell scripts as a part of a Maven build. Read up on the exec-maven-plugin to learn more.